test/parallel/test-fs-readfile-tostring-fail.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+const kMaxLength = process.binding('buffer').kMaxLength;
+
+common.refreshTmpDir();
+
+const file = path.join(common.tmpDir, 'toobig.txt');
+const stream = fs.createWriteStream(file, {
+  flags: 'a'
+});
+
+const size = 269 * 1024 * 1024;
```

Comment:
I think the value is supposed to be `process.binding('buffer').kStringMaxLength`.